### ISSUE TITLE
Please help to start!

### ISSUE THREAD

**Issue by Kaloriya** [pingpong-labs/admin#50]
how can i access to login page after installation admin package?

**Comment by gravitano** [comment]
Go to `/admin`, for example: http://domain.com/admin.

**Comment by Kaloriya** [comment]
ok, but i have this:

Not Found
The requested URL /admin was not found on this server.

What can go wrong?

**Comment by gravitano** [comment]
Are you using apache? Looks like you need to enable mod_rewrite on your
server (or local server). Try : http://domain.com/index.php/admin is it
work?

**Comment by Kaloriya** [comment]
yes, i use apache and mod_rewrite is enable, i try but has the same:

The requested URL /index.php/admin was not found on this server.

**Comment by Kaloriya** [comment]
i dont know why it start works, but i still have this one problem:

InvalidArgumentException in FileViewFinder.php line 137:
View [auth.login] not found.
in FileViewFinder.php line 137
at FileViewFinder->findInPaths('auth.login', array('/home/shaba/kievsto.com.ua/ar/resources/views')) in FileViewFinder.php line 79
at FileViewFinder->find('auth.login') in Factory.php line 151
at Factory->make('auth.login', array(), array()) in helpers.php line 614
at view('auth.login') in AuthenticatesUsers.php line 24
at AuthController->getLogin()
at call_user_func_array(array(object(AuthController), 'getLogin'), array()) in Controller.php line 256
at Controller->callAction('getLogin', array()) in ControllerDispatcher.php line 164
at ControllerDispatcher->call(object(AuthController), object(Route), 'getLogin') in ControllerDispatcher.php line 112
at ControllerDispatcher->Illuminate\Routing{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 139
at Pipeline->Illuminate\Pipeline{closure}(object(Request)) in RedirectIfAuthenticated.php line 41
at RedirectIfAuthenticated->handle(object(Request), object(Closure))
at call_user_func_array(array(object(RedirectIfAuthenticated), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 103
at Pipeline->then(object(Closure)) in ControllerDispatcher.php line 114
at ControllerDispatcher->callWithinStack(object(AuthController), object(Route), object(Request), 'getLogin') in ControllerDispatcher.php line 69
at ControllerDispatcher->dispatch(object(Route), object(Request), 'App\Http\Controllers\Auth\AuthController', 'getLogin') in Route.php line 201
at Route->runWithCustomDispatcher(object(Request)) in Route.php line 134
at Route->run(object(Request)) in Router.php line 704
at Router->Illuminate\Routing{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 139
at Pipeline->Illuminate\Pipeline{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 103
at Pipeline->then(object(Closure)) in Router.php line 706
at Router->runRouteWithinStack(object(Route), object(Request)) in Router.php line 671
at Router->dispatchToRoute(object(Request)) in Router.php line 631
at Router->dispatch(object(Request)) in Kernel.php line 236
at Kernel->Illuminate\Foundation\Http{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 139
at Pipeline->Illuminate\Pipeline{closure}(object(Request)) in VerifyCsrfToken.php line 50
at VerifyCsrfToken->handle(object(Request), object(Closure))
at call_user_func_array(array(object(VerifyCsrfToken), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline{closure}(object(Request)) in ShareErrorsFromSession.php line 54
at ShareErrorsFromSession->handle(object(Request), object(Closure))
at call_user_func_array(array(object(ShareErrorsFromSession), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline{closure}(object(Request)) in StartSession.php line 62
at StartSession->handle(object(Request), object(Closure))
at call_user_func_array(array(object(StartSession), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline{closure}(object(Request)) in AddQueuedCookiesToResponse.php line 37
at AddQueuedCookiesToResponse->handle(object(Request), object(Closure))
at call_user_func_array(array(object(AddQueuedCookiesToResponse), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline{closure}(object(Request)) in EncryptCookies.php line 59
at EncryptCookies->handle(object(Request), object(Closure))
at call_user_func_array(array(object(EncryptCookies), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline{closure}(object(Request)) in CheckForMaintenanceMode.php line 42
at CheckForMaintenanceMode->handle(object(Request), object(Closure))
at call_user_func_array(array(object(CheckForMaintenanceMode), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 103
at Pipeline->then(object(Closure)) in Kernel.php line 122
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 87
at Kernel->handle(object(Request)) in index.php line 54

**Comment by Kaloriya** [comment]
now i have this one:

Whoops, looks like something went wrong.

1/1
ErrorException in ProviderRepository.php line 66:
Undefined index: when
in ProviderRepository.php line 66
at HandleExceptions->handleError('8', 'Undefined index: when', '/home/shaba/kievsto.com.ua/ar/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php', '66', array('providers' => array('App\Providers\AppServiceProvider', 'App\Providers\EventServiceProvider', 'App\Providers\RouteServiceProvider', 'Pingpong\Admin\Providers\SupportServiceProvider', 'Pingpong\Admin\AdminServiceProvider'), 'manifest' => array('providers' => array('App\Providers\AppServiceProvider', 'App\Providers\EventServiceProvider', 'App\Providers\RouteServiceProvider', 'Pingpong\Admin\Providers\SupportServiceProvider', 'Pingpong\Admin\AdminServiceProvider'), 'eager' => array('App\Providers\AppServiceProvider', 'App\Providers\EventServiceProvider', 'App\Providers\RouteServiceProvider', 'Pingpong\Admin\Providers\SupportServiceProvider', 'Pingpong\Admin\AdminServiceProvider'), 'deferred' => array()))) in ProviderRepository.php line 66
at ProviderRepository->load(array('App\Providers\AppServiceProvider', 'App\Providers\EventServiceProvider', 'App\Providers\RouteServiceProvider', 'Pingpong\Admin\Providers\SupportServiceProvider', 'Pingpong\Admin\AdminServiceProvider')) in Application.php line 507
at Application->registerConfiguredProviders() in RegisterProviders.php line 17
at RegisterProviders->bootstrap(object(Application)) in Application.php line 203
at Application->bootstrapWith(array('Illuminate\Foundation\Bootstrap\DetectEnvironment', 'Illuminate\Foundation\Bootstrap\LoadConfiguration', 'Illuminate\Foundation\Bootstrap\ConfigureLogging', 'Illuminate\Foundation\Bootstrap\HandleExceptions', 'Illuminate\Foundation\Bootstrap\RegisterFacades', 'Illuminate\Foundation\Bootstrap\RegisterProviders', 'Illuminate\Foundation\Bootstrap\BootProviders')) in Kernel.php line 222
at Kernel->bootstrap() in Kernel.php line 117
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 87
at Kernel->handle(object(Request)) in index.php line 54
Whoops, looks like something went wrong.

1/1
ReflectionException in Container.php line 736:
Class encrypter does not exist
in Container.php line 736
at ReflectionClass->__construct('encrypter') in Container.php line 736
at Container->build('encrypter', array()) in Container.php line 626
at Container->make('encrypter', array()) in Application.php line 674
at Application->make('Illuminate\Contracts\Encryption\Encrypter') in Container.php line 837
at Container->resolveClass(object(ReflectionParameter)) in Container.php line 800
at Container->getDependencies(array(object(ReflectionParameter)), array()) in Container.php line 771
at Container->build('App\Http\Middleware\EncryptCookies', array()) in Container.php line 626
at Container->make('App\Http\Middleware\EncryptCookies', array()) in Application.php line 674
at Application->make('App\Http\Middleware\EncryptCookies') in Kernel.php line 142
at Kernel->terminate(object(Request), object(Response)) in index.php line 58
